src/utils/Parsing.fs

```diff
@@ -399,28 +412,28 @@ module internal Implementation =
                           finished <- true;                                                                             
                           valueStack.Push(ValueInfo(res, lhsPos.[0], lhsPos.[1])) 
                     | RecoverableParseError ->
-#if DEBUG
+#if __DEBUG
                           if Flags.debug then Console.WriteLine("RecoverableParseErrorException...\n");
 #endif
                           popStackUntilErrorShifted(None);
                           // User code raised a Parse_error. Don't report errors again until three tokens have been shifted 
                           errorSuppressionCountDown <- 3
                 elif kind = errorFlag then (
-#if DEBUG
+#if __DEBUG
```

Comment:
I am not that certain that we will want to include the info on a RELEASE build, for sure we haven't up to now, in some ways it just adds a new never or at least seldom used code path at least with DEBUG it will always be used when we compile DEBUG..

If you are really sure that there is value in adding a new BUILD define for this, propose a more expressive name.